src/hotspot/share/prims/jvmtiEnv.cpp

```diff
@@ -1644,120 +1644,36 @@ JvmtiEnv::GetFrameCount(JavaThread* java_thread, jint* count_ptr) {
 // java_thread - pre-checked
 jvmtiError
 JvmtiEnv::PopFrame(JavaThread* java_thread) {
```

Comment:
This change moves code with EscapeBarriers (integrated with #119) into a handshake. That does not work because object reallocation can safepoint. So the EBs are pulled out of the handshake.